config.json

```diff
@@ -2,6 +2,12 @@
   "language": "Elm",
   "slug": "elm",
   "active": true,
+  "status": {
+    "concept_exercises": true,
+    "test_runner": true,
+    "representer": true,
+    "analyzer": true
+  },
```

Comment:
That's too many changes on the config, I can't be sure of what the actual change is. I'm assuming it got sorted somehow, but could you revert that?

Reply:
I had to run configlet fmt to make the CI pass. There was a recent release for configlet so I think that’s the issue. I can open a separate PR for the config formatting. That’ll make it easier to see the actual changes.

Reply:
Oh I see, thank you for pointing it out. Please leave it like that, I checked that it's only adding what it's supposed to.

But that's strange, when I run `bin/configlet fmt` is says everything is clear, but running `bin/configlet fmt --update` on main it changes this file. Is it the same for you? I should flag that.